### ISSUE TITLE
Enable KAMI part of Thaumic Tinkerer

### ISSUE THREAD

**Issue by InferExcav** [JasonMcRay/InfiTech-1.7#685]
![default](https://cloud.githubusercontent.com/assets/11227776/15340225/fecb2468-1c90-11e6-9435-22ec6046d65a.PNG)
Can you fix this font in next updates?
Also I want to ask you to enable KAMI (Thaum Tinkerer [http://minecraft.curseforge.com/projects/thaumic-tinkerer](url) ) in the next builds with changed recepies for more hard builds wich combined with greg like this.
![image](https://cloud.githubusercontent.com/assets/11227776/15340301/5dbdc246-1c91-11e6-8db4-157d98939057.png)

**Comment by Xavion3** [comment]
So start simple, can you come up with your reasoning why KAMI should be enabled? Since a convincing enough argument could sway anyone, but for something like this a just "that could be cool" is less likely to get traction.

**Comment by InferExcav** [comment]
Kami should be added only with very hard recepies, where you need to pass titanium or iridium era to create only ichor for this mode. This will be one more branch with achievements, tools, packs, dimensions and with maybe one more way to end game if you create all stuff.

**Comment by TehSekys** [comment]
I'd second this proposition, but for a different and bit lengthier reason. I'd propose adding both Forbidden Magic and enabling TTKAMI, as the magic side of things feels lackluster at the moment.

I've completed around 90% of the pack so far, I have ore processing automated for all 7 different types of production and everything like that, a chunkloaded 10x10 chunk base, and just finished the thaumonomicon completely, and honestly, it feels pretty subpar. The actual "research" component of it is what throws me off personally, it feels too easy in relation to the goal of the pack. For a Gregtech focused pack that intends to prolong the game, there isn't near enough research at the moment. There isn't even an incentive to research *everything* in the Thaumonomicon, and TTKami would fix that. To even start working on KAMI, you need to have researched everything else in the book first, and I believe it's possible to integrate Greg's added research into this as an additional requirement. The technology side of things definitely has an endgame incentive, but magic feels like a simple requirement for a few things, mostly thaumium, and there is no real "reason" to do the endgame magic as well, which is a shame. I'm sure you guys could come up for some pretty wicked recipes to make Ichor truly frustrating to craft >:) 

Tl dr; Tech feels immersive and complete, magic feels like a simple requirement for a few things, and then you can basically forget about it unless you need something. Thaumcraft needs more research, and KAMI + Forbidden Magic would help enable it to that end.

**Comment by TehSekys** [comment]
And as the other person said, Thaumcraft without its addons feels like IC2 without Gregtech or something similar. Once you know base Thaumcraft, the research is easy. Adding KAMI and Forbidden Magic often helps alleviate that, as well as offering you powerful incentives in the late game such as extra enchantments and the KAMI gear, each of which is pretty powerful but not overpowered in it's own right.

**Comment by Xavion3** [comment]
Yeah, I've mused on some recipes for if it were to be added and it'd be tough as it'd build off the current high tier thaumcraft stuff. Adding Forbidden Magic would just make things worse as my very first idea for that is to include all seven sin shards with both the ender and nether shards.

That said, in regards to Forbidden Magic. It seems like it's significantly less useful here, it's most notable trait is compatibility between magic mods after all but with just thaumcraft that's very limited. Do we really need more for the taint and nether traits? I mean it could be interesting, it just seems like it'd be less effective here and somewhat overshadowed.

**Comment by TehSekys** [comment]
Forbidden Magic was mostly an afterthought, but it would be an effective way to bridge the gap when it comes to magical spawning of mobs, which is severely limited right now imho. For better or worse, who  knows.

But when it comes to KAMI, I feel like it should be a culmination of everything else in the pack, not just magic. KAMI armor is literally meant to be god armor lore-wise, and I'd have to say that mastering the tech that Greg offers would definitely make you fall somewhere on that spectrum. 

Top tier Gregtech resource crops, dragon eggs, maybe a few custom items dealing with exposing magic items to high-end tech things, maybe some Forestry Bee combs (yay, bee uses!) and just general high essentia costs for the infusions in the first place. 512 of several essentia types is definitely not outlandish for KAMI infusions by any means, especially since that would encourage people to learn how essentia piping works properly, instead of just stockpiling jars of the stuff beforehand. 512 is a lot, but by no means impossible.

**Comment by TehSekys** [comment]
I'd love to see a different type of difficulty where it's not required to pipe one type of essentia into essentia reservoirs, but you'd be greatly rewarded in the space department by doing so, as it would be a severe waste of space to try and use 512 of every primal for an Infusion, by using jars.

**Comment by Xavion3** [comment]
The Tablet of Necromancy is my go to solution for magical mob spawning, not the fastest but rather useful still and relatively cheap.

Thaumic Energistics lets you automate that thaumcraft and store essentia within an AE system, that's going to be beating out reservoirs for huge amounts for sure. I'd probably try and tend towards complexity over quantity of resources though.

**Comment by TehSekys** [comment]
I completely forgot Thaumic Energistics was a thing. Yeah, that completely kills everything like that. But it does mean that it makes the infusion 10 times easier if you have it set up. It definitely seems hard to make things more "complex" when Thaum and its addons are the only real "magic" in the pack. You could add some Hardcore Ender Expansion items for infusion ingredients though, some of those are rather difficult to obtain, like a bunch of tier 3 charms or the like.

**Comment by TehSekys** [comment]
Which would ensure that they have to have progressed in Hardcore Ender Expansion to craft the charms / items in the first place, meaning progression on two fronts, which seems pretty complex to me as a starter position.

**Comment by Xavion3** [comment]
So, KAMI is being enabled. Already done a decent amount, included various bits and bobs from HEE in the recipes. Any suggestions as to things you'd like to see with modified recipes there will be considered though. A couple of random examples from recipes so far are making awakened tools require enhanced TNT from HEE instead of normal TNT, the gem tools/armour to require high tier gems of the right colour as opposed to just diamonds, and the foci being modified to IT2 style with lens + 4 rings, the rings made of ichorium naturally.
